### ISSUE TITLE
Add support for CW-Husky

### ISSUE THREAD

**Issue by vogelpi** [lowRISC/ot-sca#50]
Our capture scripts still use the CW-Lite scope. We should switch to CW-Husky which has higher capture performance.

**Comment by colinoflynn** [comment]
I've also started adding some documentation here: https://github.com/newaetech/ot-sca/blob/cwhusky-doc/doc/getting_started_bergen.md

I'm using a separate branch for now to avoid polluting the active PR. If the documentation is done before that PR gets merged I'll move it over, but for now the commits will be a bit messy on the `cwhusky-doc` branch. However I wanted to get some notes out there ASAP, and if there is anything else useful please feel free to push into that.

**Comment by m-temp** [comment]
> We have planned to restructure a bit the directory structure and with that should also modify cw305. No need to change that in your PR.

That's something we should definitively do.
Additionally, there seems to be some miss-configuration (on my side?) between git and git lfs:
When using the binaries from e64e76dfd09f8ac23ca8ba42465420e63e3c5381 I get `ValueError: Invalid file size: 41`
An additional `git lfs pull` provides new (?) different (?) binaries, that are marked as `modified`. However, I am not able to attack them. (0 to2 diffs recovered) Are they masked?

>I would expect the Husky to give better performance but the variation with the bitstream is big.

Building my own unmasked `aes_serial_fpga_cw310` from  lowRISC/opentitan@1b73f8090033839b73de6b5954702137d723948e and using a scope_gain of 35, I get the following:
```
./sweep_ceca.py -f projects/opentitan_simple_aes.cwp -s 3 -w 16 -d output -a 518 527
#traces
          key recovered
            #diffs recovered
    10000 0   0
   100000 0  14
   200000 0  52
   300000 0  80
   400000 0  93
   500000 1 104
   600000 1 110
   700000 1 112
   800000 1 111
   900000 1 114
  1000000 1 114
  1100000 1 116
  1200000 1 118
  1300000 1 119
  1400000 1 117

```
(Ran out of memory for more traces)

However, I have the impression that the measurements are not very reliable/reproducible. I have to explore this a little bit more.

**Comment by vogelpi** [comment]
I am closing this issue as the PR for enabling CW-Husky has been merged a while ago already (#56 ). Btw, I've finally managed to try it out and it's great. I am really happy to have it, thanks @colinoflynn and @jpcrypt !
